dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ItemRestRepository.java

```diff
@@ -138,14 +140,31 @@ public ItemRest findOne(Context context, UUID id) {
     @PreAuthorize("hasPermission(#id, 'ITEM', #patch)")
     protected void patch(Context context, HttpServletRequest request, String apiCategory, String model, UUID id,
                          Patch patch) throws AuthorizeException, SQLException {
-        patchDSpaceObject(apiCategory, model, id, patch);
+
+        Item item = itemService.find(obtainContext(), id);
+        if (item == null) {
+            throw new ResourceNotFoundException(apiCategory + "." + model + " with id: " + id + " not found");
+        }
+        if (item.getTemplateItemOf() != null) {
+            throw new DSpaceBadRequestException("The given ID resolved to a template item");
```

Comment:
I'd recommend updating this error message slightly to return the ID (as it's more useful to reference the problematic object).  So, something like `"The ID " + id + " resolved to a template item"`